tests/test_server_proto.py

```diff
@@ -685,6 +685,27 @@ async def test_server_proto_args_07(self):
             await self.con.fetchone(
                 'select schema::Object {name} filter .id=$id', id='asd')
 
+    async def test_server_proto_args_08(self):
```

Comment:
I'd add a similar test for named args also

Reply:
You had a very good point: indeed, it fails with another kind of error with named arguments.

```
File "edgedb/protocol/protocol.pyx", line 998, in edgedb.protocol.protocol.SansIOProtocol.encode_args
RuntimeError: expected named arguments, got positional arguments
```

Reply:
That RuntimeError should be changed to `errors.InterfaceError`

Reply:
This PR is becoming stale, let's revive it @RobertoPrevato